indico/modules/events/editing/client/js/components/FileManager/index.jsx

```diff
@@ -151,10 +151,14 @@ function FileType({uploadURL, fileType, uploads}) {
                 </Translate>
               }
               trigger={
-                <Segment>
+                <Message
+                  onDismiss={() => {
```

Comment:
`{() => actions....(...)}` doesn't fit?

Reply:
That was how I did it at first but prettier is complaining that way

Reply:
weird.. what did it complain about?